main.py

```diff
@@ -91,220 +99,353 @@ async def startDriver(): #start a new driver and log in to gapps
 #to tell blocked thread to stop monkeying
 end = False
 
-@client.event
-async def on_message(message):
+#drop a link from the queue
+@slash.slash(
+  name="drop",
+  description="Drop a link",
+  options=[
+    manage_commands.create_option(
+      name="link",
+      description="The Meet code/link or Zoom link to be dropped",
+      option_type=3,
+      required=True
+    )
+  ],
+  guild_ids=[guild_id]
```

Comment:
what is the purpose of this?

Reply:
`guild_ids` is required to avoid a ten minute delay in command activation.

Reply:
what is guild_ids

Reply:
The guild ids that the command will be active in

Reply:
As such, it is required that the guild ID be provided in `settings.json`.